Grocery App/Controller/MainViewController.swift

```diff
@@ -7,19 +7,26 @@
 
 import UIKit
 
-class MainViewController: MyCustomTabBarController {
+class MainViewController: UITabBarController, UITabBarControllerDelegate {
     
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
         
+        self.delegate = self
+        self.selectedIndex = 2
         ConfigureViewController()
         
-    }
+        }
     
     func ConfigureViewController(){
```

Comment:
Method name should start with small character.
Try to avoid fetching tab bar items using direct index. 
Traverse tab bar items and then apply condition for setting title and image.

Reply:
Traversed tabbar  items using for loop and then set title and image.